habitat_sim/robots/mobile_manipulator.py

```diff
@@ -193,9 +196,30 @@ def update(self) -> None:
         # Update the cameras
         for sensor_name in self._head_sensor_names:
```

Comment:
This repeated code has bad code smell.

Reply:
Agreed, I think this would be fixed by a better robot camera API.